general/v0.1/provider_specifications.md

```diff
@@ -61,37 +60,6 @@ paragraph description in English that can be used by fediverse software
 to explain the capabilities to fediverse server administrators. It MAY include
 translations into other languages as well. 
 
-### OAuth 2.0 Scopes
```

Comment:
I think it'd be worth still having FASPs register certain capabilities or roles, but these would just be stored in relation to the install in the fediverse software

Reply:
Which is exactly why we define "capabilities". That was another problem with OAuth in fact, because it was not at all clear how scopes should map to capabilities and if they even need to correspond to each other in the first place.